src/app/screens/ledger/importLedgerAccount/index.tsx

```diff
@@ -406,7 +438,8 @@ function ImportLedger(): JSX.Element {
           btcPublicKey: btcCreds?.publicKey || '',
           ordinalsPublicKey: ordinalsCreds?.publicKey || '',
           accountType: 'ledger',
-          accountName: `Ledger Account ${newAddressIndex + 1}`,
+          accountName: `Ledger Account ${newAccountId + 1}`,
+          deviceAccountIndex: getDeviceNewAccountIndex(masterPubKey || masterFingerPrint),
```

Comment:
type error here

Reply:
Added the related PR to fix it: https://github.com/secretkeylabs/xverse-core/pull/198